pkg/continuoustest/write_read_series.go

```diff
@@ -22,6 +22,14 @@ const (
 	metricName    = "mimir_continuous_test_sine_wave"
 )
 
+var (
+	// We use max_over_time() with a 1s range selector in order to fetch only the samples we previously
+	// wrote and ensure the PromQL lookback period doesn't influence query results. This help to avoid
+	// false positives when finding the last written sample, or when restarting the testing tool with
+	// a different number of configured series to write and read.
+	queryMetricSum = fmt.Sprintf("sum(max_over_time(%s[1s]))", metricName)
```

Comment:
Maybe `last_over_time` would be a semantically better approach here?

Reply:
Right. Don't see much different tho. Will keep `max_over_time` since I've already tested it, so I can move on.